selfdrive/ui/qt/widgets/controls.h

```diff
@@ -127,3 +126,28 @@ class ParamControl : public ToggleControl {
 private:
   Params params;
 };
+
+class ListWidget : public QWidget {
+  Q_OBJECT
+ public:
+  explicit ListWidget(QWidget *p = nullptr) : QWidget(p), layout_(this) {
```

Comment:
```suggestion
  explicit ListWidget(QWidget *parent = 0) : QWidget(parent), layout_(this) {
```
let's keep it consistent